.storybook/main.js

```diff
@@ -62,13 +67,24 @@ module.exports = /** @type {Omit<StorybookConfig,'typescript'|'babel'>} */ ({
 
     // internal monorepo custom addons
 
-    /**  @see ../packages/react-components/react-storybook-addon */
+    /**  {@link file://./../packages/react-components/react-storybook-addon/package.json} */
     loadWorkspaceAddon('@fluentui/react-storybook-addon', { tsConfigPath }),
-    loadWorkspaceAddon('@fluentui/react-storybook-addon-export-to-sandbox', { tsConfigPath }),
+    /** {@link file://./../packages/react-components/react-storybook-addon-export-to-sandbox/package.json} */
+    loadWorkspaceAddon('@fluentui/react-storybook-addon-export-to-sandbox', {
+      tsConfigPath,
+      /** @type {import('../packages/react-components/react-storybook-addon-export-to-sandbox/src/public-types').PresetConfig} */
```

Comment:
instantiation of generic in vanilla js 

in TS the syntax looks like this: `loadWorkspaceAddon<import('../packages/react-components/react-storybook-addon-export-to-sandbox/src/public-types').PresetConfig}>(...)`